source/handlers/desktopparser.d

```diff
@@ -204,11 +201,18 @@ Component parseDesktopFile (GeneratorResult gres, string fname, string data, boo
         if (key.startsWith ("Name")) {
             immutable val = getValue (df, key);
             checkDesktopString (key, val);
-            cpt.setName (val, locale);
+            /* run backend specific hooks */
+            auto translations = gres.pkg.processDesktopFile (df, val);
```

Comment:
I just wondered: Should this method have a more specific name than "processDesktopFile"? Something like "getDesktopFileKey" or "getDesktopFileTranslation"?